### ISSUE TITLE
Component: Buttons

### ISSUE THREAD

**Issue by oliverschuerch** [swisspost/design-system#580]
Implement a buttons web-component.

Properties:
- [ ] variant (string: primary, light, dark, link, etc.)
- [ ] size (string: sm, md, lg, etc.)
- [ ] tag (string: button, a, input)
- [ ] type (string: button, submit, etc.)
- [ ] href (string)
- [ ] target (string: _self, _blank, etc.)
- [ ] active (boolan)
- [ ] disabled (boolean)
- [ ] block (boolean)
- [ ] animated (boolean)
- [ ] icon (boolean)

Slots:
- [ ] default

Events:
- [ ] Click

**Comment by oliverschuerch** [comment]
Warning:
Some SCSS methods/selectors are breaking the import of the button styles in the Stencil Component "post-button":

`@extend %btn-default;` placeholder
`@include utilities.not-disabled-focus-hover()` mixin

**Comment by oliverschuerch** [comment]
@gfellerph please see my previous post.
I could fix this issues, so the button styles should work for the post-button component as well.
What do you think, how should we act to get rid of this issues?

**Comment by oliverschuerch** [comment]
This Ticket is blocked by #603

**Comment by gfellerph** [comment]
Specification needs a discussion.

**Comment by gfellerph** [comment]
Closing as out of scope
